tests/components/recorder/test_init.py

```diff
@@ -944,8 +944,6 @@ async def test_saving_event_invalid_context_ulid(
             )
         }
 
-    assert "invalid" in caplog.text
```

Comment:
The invalid logging did not turn out to be useful so it was removed when it was moved to the lib